src/OneBlinkFormBase.tsx

```diff
@@ -147,6 +147,10 @@ function OneBlinkFormBase({
   // #region Unsaved Changed
 
   const history = useHistory()
+  const isKeyUser = React.useMemo(() => {
+    const token = authService.getUserToken()
```

Comment:
This should be using the hook `useAuth` which returns a property `isUsingFormsKey`